docs-chef-io/content/supermarket/supermarket_upgrade.md

```diff
@@ -13,93 +13,163 @@ publishDate = 2022-01-03
     weight = 25
 +++
 
-
 ## Upgrade Matrix
+  If running Supermarket 4.2, you can upgrade directly to the latest releases of Supermarket 4.3. If you are running a release with version less than 4.2 you must perform a stepped upgrade as outlined below.
```

Comment:
The markdown in this file is bad. It looks like the docs-lint GitHub action didn't run. Could you look into it--by which I mean fix it? The file is https://github.com/chef/supermarket/blob/main/.github/workflows/docs-lint.yml . Once fixed, the tests will let you know what to correct.

Reply:
Sure. let me have a look.

Reply:
Thanks for highlighting @kagarmoe

Reply:
@kagarmoe  The issues with this particular file has been resolved. Pls have a look.
Although I'll raise a separate issue with docs-lint to fix the issue with marklint plugin of github.